spec.html

```diff
@@ -25000,7 +24999,7 @@
         <emu-alg>
           1. Let _x_ be ? thisNumberValue(*this* value).
           1. Let _f_ be ? ToInteger(_fractionDigits_). (If _fractionDigits_ is *undefined*, this step produces the value 0.)
-          1. If _f_ &lt; 0 or _f_ &gt; 20, throw a *RangeError* exception. However, an implementation is permitted to extend the behaviour of `toFixed` for values of _f_ less than 0 or greater than 20. In this case `toFixed` would not necessarily throw *RangeError* for such values.
+          1. If _f_ &lt; -20 or _f_ &gt; 100, throw a *RangeError* exception.
```

Comment:
Well, the definition follows mathematically pretty cleanly; I don't see why we should especially throw on 0. Anyway, I also can't think of a time when I'd want to use a negative number, and would be fine with throwing on negative offsets (and, I probably wouldn't use any of these functions, and use `Intl.NumberFormat` instead).

I'll see if we can get more from Waldemar about the motivation for these semantics in his implementation.